vmm/src/seccomp_filters.rs

```diff
@@ -236,6 +241,7 @@ pub fn vmm_thread_filter() -> Result<SeccompFilter, Error> {
             allow_syscall(libc::SYS_mremap),
             allow_syscall(libc::SYS_munmap),
             allow_syscall(libc::SYS_nanosleep),
+            #[cfg(target_arch = "x86_64")]
```

Comment:
I'm confused, does then open syscall not exist on aarch64?

Reply:
Checked the docs of [rust-lang/libc](https://docs.rs/libc/0.2.70/libc/constant.SYS_open.html) and source code, it is indeed unavailable on aarch64.

This is also confusing me.

Reply:
It uses the `openat()` syscall instead.